Client/Assets/Images.xcassets/fxHomeHeaderLogoText.imageset/Contents.json

```diff
@@ -1,7 +1,7 @@
 {
   "images" : [
     {
-      "filename" : "fxLogoText.pdf",
+      "filename" : "fxHomeHeaderLogoText.svg",
```

Comment:
Don't we prefer pdf over svg? cc @thatswinnie

Reply:
I was just going to comment on this. We use PDFs, not SVGs, so we should update the asset. :)